docs/reference/api_references.rst

```diff
@@ -299,12 +299,17 @@ cms.app_base
 
     .. method:: get_menus(page, language, **kwargs)
 
-            Returns the menus for the apphook instance, eventually selected according
+        .. versionadded:: 3.3
+            ``CMSApp.get_menus`` accepts page, language and generic keyword arguments:
+            you can customize this function to return different list of menu classes
+            according to the given arguments.
+
+            Returns the menus for the apphook instance, selected according
             to the given arguments.
 
-            By default it returns the menus assigned to :py:attr:`CMSApp._menus`
+            By default it returns the menus assigned to :py:attr:`CMSApp._menus`.
 
-            If no page and language si provided, this method **must** return all the
+            If no page and language are provided, this method **must** return all the
             menus used by this apphook. Example::
```

Comment:
The thing I need to check whether this phrase is true https://github.com/evildmp/django-cms/blob/99edf3ab22f7f03dddba50663982bd1957158a1f/cms/app_base.py#L87